samples/fpga/gemm_systolic_vectorized.py

```diff
@@ -0,0 +1,430 @@
+# Copyright 2019-2020 ETH Zurich and the DaCe authors. All rights reserved.
+# Computes C = AB + C
+# This sample build on matrix_multiplication_systolic by adding
+# vectorization. The systolic arrays used data type depends on the used vectorization width
+# (e.g., float16 for vec_width = 16)
+
+import argparse
+import dace
+import numpy as np
+import pdb
+import select
+import sys
+
+N = dace.symbol("N")
+K = dace.symbol("K")
+M = dace.symbol("M")
+P = dace.symbol("P")
```

Comment:
You could also make `W` a symbol (we used to not support this, but it should work now)?

(if it doesn't work, forget about it)